### PR TITLE
add to grayscale test prototype

### PR DESCRIPTION
**Pull request by edgarriba**
#### Changes

Prototype test for the grayscale to rgb

#### Type of change

- [ ] 📚  Documentation Update
- [x] 🧪 Tests Cases
- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🔬 New feature (non-breaking change which adds functionality)
- [ ] 🚨 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📝 This change requires a documentation update

#### Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Did you update CHANGELOG in case of a major change?

### CODE REVIEW

File: test/color/test_gray_core.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            _shape = (num_channels, *shape)
+            channels_order = ChannelsOrder.CHANNELS_FIRST
+        else:
+            _shape = (*shape, num_channels)
+            channels_order = ChannelsOrder.CHANNELS_LAST
+        image_gray_data = _make_random_image_u8(_shape, device, dtype, backend)
+        image_gray = Image(
+            data=image_gray_data,
+            pixel_format=PixelFormat(color_space=ColorSpace.GRAY, bit_depth=8),
+            layout=ImageLayout(
```
Comment (johnnv1) [inline test/color/test_gray_core.py:48]: this is looking too complex for an API to replace a simple tensor -- maybe we can pass the `channels_order` into the Image constructor, and from this construct what the layout is

Comment (johnnv1) [reply test/color/test_gray_core.py:48]: Also for keras core we should consider using the `keras_core.config.image_data_format()` as the default value for the layout

https://github.com/keras-team/keras-core/blob/v0.1.3/keras_core/backend/config.py#L106

Comment (johnnv1) [reply test/color/test_gray_core.py:48]: Also, use this flag to generate the inputs under the test suite, setting it like the actual device/dtype fixture (https://keras.io/keras_core/api/utils/config_utils/#setimagedataformat-function)

Comment (edgarriba) [reply test/color/test_gray_core.py:48]: sure we iterate that

File: kornia/image/image.py [hunk 2]
```diff
@@ -160,6 +164,12 @@ def channels_order(self) -> ChannelsOrder:
         """Return the channels order."""
         return self.layout.channels_order
 
+    def get_channel(self, idx: int):
```
Comment (johnnv1) [inline kornia/image/image.py:167]: maybe add a note here to make it work with str -- like:

```python
r = image.get_channel('R')
# maybe in the future
r, b, g = image.get_channel('RGB')
```

**Review by johnnv1 — APPROVED** [review]

File: kornia/image/image.py (hunk 2 above)
Comment (edgarriba) [reply kornia/image/image.py:167]: i think we are gonna drop for now the extension of image api at least for the first iteration

**Comment by edgarriba** [comment]
closing for project inactivity

**Pull request closed without merging**